app/controllers/posts_controller.rb

```diff
@@ -210,6 +210,13 @@ def update
       edit_reason: params[:post][:edit_reason]
     }
 
+    # Check if there is no edit conflict caused by two users saving different
+    # versions of the post.
```

Comment:
The code itself explains the behavior quite clearly so I'm not sure if we really need this comment here.